tests/dragonfly/cluster_test.py

```diff
@@ -1567,6 +1567,135 @@ async def test_cluster_replication_migration(
     assert await seeder.compare(r1_capture, r2_node.instance.port)
 
 
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes"})
+async def test_run_replication_during_migration(
```

Comment:
nit
rename test_start_replication_during_migration
I also think its a good practice to add a short comment at the beginig of test what is the goal of the test and for complicated test with lots of steps add a breef steps description